vmdb/app/models/tag.rb

```diff
@@ -104,4 +107,29 @@ def self.filter_ns(tags, ns)
   def ==(comparison_object)
     super || name.downcase == comparison_object.to_s.downcase
   end
+
+  def category
+    @category ||= Classification.find_by_name(name_path.split('/').first, nil)
+  end
+
+  def show
+    category && category.show
+  end
+
+  def categorization
+    @categorization ||= begin
+      !show ? {} : {
+        "name"         => classification.name,
+        "description"  => classification.description,
+        "category"     => {"name" => category.name, "description" => category.description},
+        "display_name" => "#{category.description}: #{classification.description}"
+      }
```

Comment:
ternary should not span multiple lines...let's just make it a normal if/else.  Also, since it's a single expression, you don't need the begin/end